prover/config.yml

```diff
@@ -17,3 +17,5 @@ metrics_port: 9100
 enable_dangerous_logging: false
 enable_debug_checks: false
 enable_federated_jwks: false
+disable_exp_claim_in_future_check: false
```

Comment:
Please rename `disable_exp_claim_in_future_check ` to `enable_jwt_exp_check` for added clarity: (1) when I first read this, I mis-interpreted as the `exp_date` and (2) easier to reason when all configuration options are `enable_*`

Reply:
If you feel it is important to mention what the validation does (I think it is implicitly stated in "JWT `exp` validation"), rename to `enable_jwt_exp_not_in_the_past_check` (because the check ensures that the expiration date in the JWT is not in the past).